pkg/kev/skaffold.go

```diff
@@ -60,6 +71,22 @@ func LoadSkaffoldManifest(path string) (*SkaffoldManifest, error) {
 	return s, yaml.Unmarshal(data, &s)
 }
 
+// AddProfiles injects kev profiles to existing Skaffold manifest
+// Note, if profile name already exists in the skaffold manifest then profile won't be added
+func AddProfiles(path string, envs []string, includeAdditional bool) (*SkaffoldManifest, error) {
```

Comment:
Looking around the code `includeAdditional` is always `true` ... and `NewSkaffoldManifest`  runs `AdditionalProfiles()`.

Is there a case where we wouldn't want to run `AdditionalProfiles()`?

Reply:
This is useful primarily in testing. Might add additional flag to control whether to include additional (helper) profiles - which currently are included by default - to the manifest in the future PR.